GitCommands/Git/GitModule.cs

```diff
@@ -2725,9 +2725,19 @@ public bool IsDirtyDir()
                 : null;
         }
 
+        /// <summary>
+        /// Get the file contents in HEAD for the file
+        /// </summary>
+        /// <param name="file">The Git status item</param>
+        /// <returns>async Task with contents</returns>
```

Comment:
```suggestion
        /// <returns>An awaitable task with the requested file content.</returns>
```

Reply:
As discussed below (currently) changed to
```suggestion
        /// <returns>An awaitable task with the requested file contents.</returns>
```